model/companyModel.js

```diff
@@ -5,13 +5,13 @@ const LocationSchema = require('../util/schemas/locationSchema');
 const companySchema = Mongoose.Schema({
     name: {
         type: String,
-        required: true,
+        required: [true, 'Company name must be specified'],
         trim: true,
-        minlength: [1, 'Company name cannot be left empty']
+        minlength: [1, 'Company name must be at least 1 character long']
```

Comment:
I think that every property that has a `minlength` check should also have a `maxlength` check. Set values as you see fit.

Reply:
Okay, I'll base the max lengths on this:
https://stackoverflow.com/questions/20958/list-of-standard-lengths-for-database-fields